src/vpp_plugins/vpp-chaperone/VppConfigManager.cpp

```diff
@@ -1625,12 +1630,14 @@ void VppConfigManager::setTctblEntry (VppClient &vppClient,
 
 void VppConfigManager::doCpePolicerConfig (VppClient &vppClient,
                                            const std::string &interface,
-                                           const folly::dynamic &policers)
+                                           const folly::dynamic &policers,
+                                           bool policerPresent)
 
 {
   u32 interfaceIndex = (u32)~0;
   auto ifaceMap = vppClient.getIfaceToVppIndexMap ();
   auto ifaceMapIter = ifaceMap.find (interface);
+  VppClient::PolicerConfig policerConfig;
```

Comment:
Move this back into the for loop - why did you move it to the top?

Reply:
Addressed Review Comment.